modules/core/nuxeo-core-management-jtajca/src/test/resources/ds-contrib-with-fatal.xml

```diff
@@ -2,11 +2,10 @@
 <component name="org.nuxeo.runtime.jtajca.management.test.ds">
   <extension target="org.nuxeo.runtime.datasource" point="datasources">
     <datasource name="jdbc/canrollback" driverClassName="${nuxeo.test.vcs.driver}"
-      maxPoolSize="20" minPoolSize="5">
+      maxTotal="20" minTotal="5">
```

Comment:
Why changing names? maxPoolSize sounds better than maxTotal.

Reply:
I'm re-aligning with the official standard names of Apache DBCP (and therefore also Tomcat). The old names are still kept for compat though (see `PooledDataSourceRegistry.createPooledDataSource`)